pkgs/misc/drivers/hplip/default.nix

```diff
@@ -68,7 +68,7 @@ python2Packages.buildPythonApplication {
     pkgconfig
   ];
 
-  pythonPath = with python2Packages; [
+  pythonPath = with python3Packages; [
     dbus
     pillow
     pygobject2
```

Comment:
Does this really use old pygobject bindings? I feel like this dependency would be really out of date (something something 2010)

Reply:
There's a distros.dat file that suggests you can use different version of that library `pygobject3`.

Reply:
you're good :)

Reply:
@grahamc Sometime I amaze myself. (reading back my comments, I wonder was it really me who typed it?)

Reply:
happens to me all the time as well :(